Update privacy policy: new blog platform Ghost

philipecosia opened the pull request:
Update to the privacy policy section 6b. Blog functions to reflect switch from Tumblr to Ghost.

philipecosia committed:
Update privacy policy: new blog platform Ghost

Update to the privacy policy section 6b. Blog functions to reflect switch from Tumblr to Ghost.

DoHe commented on en.md:
```diff
@@ -72,15 +72,9 @@ Microsoft/Bing is certified under the “Privacy Shield” US-EU data protection
 We provide a blog in which we publish various posts on topics about our activities in order to circulate our website and its contents on the internet and to increase our visitor numbers. 
 
-We use the “Tumblr” blogging platform to operate the blog. The legal basis for this processing of your personal data is Art. 6 Abs. 1 lit. f of the General Data Protection Regulation. Tumblr is operated by Tumblr Inc., 35 East 21st St, Ground Floor, New York, NY 10010, USA.
+We use the “Ghost” blogging platform to operate the blog. Ghost does not have access to any of your personal information. For information on data protection with this third-party supplier, go to: https://ghost.org/privacy/.
```
Wasn't IP a personal information? They do have access to that obviously.

philipecosia replied:
@DoHe Good point. I updated it.

philipecosia commented:
Important: This is only to be merged once a) reviewed b) ghost is live for users and c) the German and French version are also updated.

@DoHe Could you help me out with this. From what I can tell looking at Ghost's privacy policy and their DPA even if we don't host it ourselves in the beginning they don't have access to our users personal information. https://ghost.org/privacy/ https://ghost.org/dpa/ But I'm not sure.

philipecosia committed:
Ghost has access to IP address 

as long as we don't host it ourselves

DoHe commented:
They seem to mostly talk about data they themselves gather from us, not so much about the data they might collect when hosting a blog.

philipecosia committed:
Update German privacy policy: New Blog platform Ghost

Commit:
Merge remote-tracking branch 'origin/philipecosia-patch-3' into philipecosia-patch-2

Commit:
First changes to fr.md file

DoHe commented:
@philipecosia you can just post a comment and mention me if you want me to review this again

philipecosia committed:
Adding note about 3rd party embeds blog

I took the part about 3rd party embeds largely from the Medium privacy policy. https://medium.com/policy/medium-privacy-policy-f03bf92035c9 I am not going into detail what data is shared with these 3rd parties etc. I hope keeping it this vague is legally fine.

The second question is about whether or not we need to add anything about the links we have to social media on the blog, on the info pages, on the index page and in other places. As far as I can tell we don't need to add anything to our privacy policy as long as these are just simple links and not plugins or the like. But I was not able to find any final confirmation of that.

philipecosia committed:
Added note on 3rd party embeds to german version

philipecosia committed:
Updated date on en version

philipecosia committed:
Change "hosted" to "hosting"

Less confusing wording.

philipecosia committed:
Updated changedate

philipecosia committed:
Added FR translations + new change date

LostCrew commented on fr.md:
```diff
@@ -231,4 +226,4 @@ Pour faire valoir une opposition ou une révocation, vous pouvez contacter notre
 
 ---
 
-_Date de la déclaration sur la protection des données : 08/05/2018_
```
Do we want to automatize this? A minuscole and inobtrusive solution could be found

knee-guts reviewed (approved):
🔌
